### PR TITLE
deprecate 3.9 prow

### PR DESCRIPTION
**Pull request by mjudeikis**
remove 3.9 OSA tests 

cc: @kargakis

### CODE REVIEW

**Comment by mjudeikis** [comment]
Did I miss anything in this repo?

**Comment by mjudeikis** [comment]
cc: @openshift/sig-azure

File: projects/azure/acs-engine/test-image-builds/build-3.9.yaml [hunk 1]
```diff
@@ -1,26 +0,0 @@
-kind: BuildConfig
```
Comment (kargakis) [inline projects/azure/acs-engine/test-image-builds/build-3.9.yaml]: If you are removing this, then we also need a PR to remove the 3.9 jobs from acs-engine.

Comment (kargakis) [reply projects/azure/acs-engine/test-image-builds/build-3.9.yaml]: Search for all job that use `registry.svc.ci.openshift.org/azure/acs-engine-tests:v3.9` and remove them.

**Comment by kargakis** [comment]
You are missing `test_pull_request_origin_extended_conformance_azure_39` from `ci-operator/jobs/openshift/origin/openshift-origin-presubmits.yaml` (I know it's not a ci-operator job ..)

**Comment by mjudeikis** [comment]
@kargakis done

**Pull request merged**